CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Change
 * Allow client to request model with string features used as pandas categoricals during training
+* Add support of scikit-learn models
```

Comment:
I think the first change isn't merged yet^^ (Arnaud's MR, right?)

Can you update `VERSION` file too so we can tag the new version just after the merge?